### PR TITLE
Update editors

### LINKED ISSUE #548: Update editor plugins with `require` as special syntax
I will also need to add `require` as special syntax and then update the plugin.

I might not have time to do it soon, so could you please create a ticket and assign me, so that I do not forget?

_Originally posted by @xsebek in https://github.com/swarm-game/swarm/pull/533#discussion_r912895528_

### PR DESCRIPTION
**Pull request by xsebek**
- update the highlighting for `require` in editors
- bump VSCode plugin version to 0.0.5
- update VSCode plugin changelog
- update the VSCode plugin image link
- copy LICENSE to the VSCode folder (`vsce` was complaining about it)
- rename the team in LICENSE file to Swarm
- closes #548

### CODE REVIEW

**Comment by xsebek** [comment]
@byorgey how should I push tags? :thinking: Should they go to this branch or should I push them directly to `main`?

Or maybe it would be best to use GitHub UI in releases since I am such a noob with git tags. :smile:

**Comment by byorgey** [comment]
@xsebek tags are like branches that never move. That is, a tag is associated with a particular commit.  So if you tagged a commit in this branch it would always stat on that commit, it would never end up in main when this branch is merged.

File: editors/vscode/LICENSE
Comment (byorgey) [inline editors/vscode/LICENSE:17]: Swarm?

File: editors/vscode/LICENSE
Comment (byorgey) [inline editors/vscode/LICENSE:1]: Makes more sense to put your name, I think.

**Review by byorgey — APPROVED** [review]

File: editors/vscode/LICENSE
Comment (xsebek) [reply editors/vscode/LICENSE:17]: It's what's in the LICENSE file that I just copied. :man_shrugging:

File: editors/vscode/LICENSE
Comment (xsebek) [reply editors/vscode/LICENSE:1]: It's just a copy-paste of the Microsoft template with generated TextMate grammar and keywords generated from the swarm binary, so I don't think it matters. :smile:

If we ever want to change the license for some reason, keeping it simple and not making it look like you have to get my signed permission could save us some headache. Licenses are complicated and I dread lawyers. :fearful:

Comment (byorgey) [reply editors/vscode/LICENSE:1]: OK, fair enough!

**Comment by byorgey** [comment]
@xsebek can we merge this (after updating `Disco` to `Swarm` of course)?  Or was there still more you wanted to do?

**Comment by xsebek** [comment]
@TristanCacqueray I think the current rule for Mergify might prevent PRs not changing Haskell from merging?

It seems like it is waiting for CI to run but it never will. :confused:

**Pull request merged**
